tests/markdown_empty/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`empty.md 1`] = `
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`;
```

Comment:
Every test that changed are regressions except for this one

Reply:
I'll fix it later, could you add a WIP label or something?

Reply:
Sure! Take your time